JS/Planes/PassengerPlane.js

```diff
@@ -2,21 +2,18 @@ const Plane = require('./Plane');
 
```

Comment:
Please rename the folder with plane Classes: Planes -> planes.

Reply:
Done